[2024-09-19T21:39:44.912+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-19T21:39:44.932+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nba_data_pipeline.run_data_processing manual__2024-09-19T21:38:21.389569+00:00 [queued]>
[2024-09-19T21:39:44.942+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nba_data_pipeline.run_data_processing manual__2024-09-19T21:38:21.389569+00:00 [queued]>
[2024-09-19T21:39:44.943+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 4
[2024-09-19T21:39:44.960+0000] {taskinstance.py:2888} INFO - Executing <Task(DockerOperator): run_data_processing> on 2024-09-19 21:38:21.389569+00:00
[2024-09-19T21:39:44.968+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'nba_data_pipeline', 'run_data_processing', 'manual__2024-09-19T21:38:21.389569+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/nba_data_pipeline.py', '--cfg-path', '/tmp/tmpwfur7mtl']
[2024-09-19T21:39:44.972+0000] {standard_task_runner.py:105} INFO - Job 9: Subtask run_data_processing
[2024-09-19T21:39:44.974+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=661) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-19T21:39:44.975+0000] {standard_task_runner.py:72} INFO - Started process 662 to run task
[2024-09-19T21:39:45.024+0000] {task_command.py:467} INFO - Running <TaskInstance: nba_data_pipeline.run_data_processing manual__2024-09-19T21:38:21.389569+00:00 [running]> on host 19c433e2f996
[2024-09-19T21:39:45.142+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='nba_data_pipeline' AIRFLOW_CTX_TASK_ID='run_data_processing' AIRFLOW_CTX_EXECUTION_DATE='2024-09-19T21:38:21.389569+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-19T21:38:21.389569+00:00'
[2024-09-19T21:39:45.143+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-19T21:39:45.193+0000] {docker.py:367} INFO - Starting docker container from image us-west1-docker.pkg.dev/nba-fantasy-ml/nba-data-pipeline-repo/data-processing:latest
[2024-09-19T21:39:45.203+0000] {docker.py:375} WARNING - Using remote engine or docker-in-docker and mounting temporary volume from host is not supported. Falling back to `mount_tmp_dir=False` mode. You can set `mount_tmp_dir` parameter to False to disable mounting and remove the warning
[2024-09-19T21:39:46.059+0000] {docker.py:438} INFO - 2024-09-19 21:39:46,057 - INFO - Starting data processing...
[2024-09-19T21:39:46.063+0000] {docker.py:438} INFO - 2024-09-19 21:39:46,062 - INFO - Database connected.
[2024-09-19T21:39:46.064+0000] {docker.py:438} INFO - 2024-09-19 21:39:46,062 - INFO - Cleaning and preprocessing data...
[2024-09-19T21:39:46.101+0000] {docker.py:438} INFO - 2024-09-19 21:39:46,099 - INFO - Validation passed.
[2024-09-19T21:39:46.101+0000] {docker.py:438} INFO - 2024-09-19 21:39:46,099 - INFO - Assigning unique IDs for players and games...
[2024-09-19T21:39:46.176+0000] {docker.py:438} INFO - 2024-09-19 21:39:46,175 - INFO - Inserting player stats into database...
[2024-09-19T21:39:46.273+0000] {docker.py:438} INFO - 2024-09-19 21:39:46,271 - INFO - Data processing completed successfully.
[2024-09-19T21:39:46.274+0000] {docker.py:438} INFO - 2024-09-19 21:39:46,271 - INFO - Database connection closed.
[2024-09-19T21:39:46.274+0000] {docker.py:438} INFO - 2024-09-19 21:39:46,271 - INFO - Successfully processed player_box_scores/2023-24/nba_player_stats_2023-24_2023-10-24_to_2023-10-25_2024-09-19_21-39-43.csv
[2024-09-19T21:39:46.575+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-19T21:39:46.576+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=nba_data_pipeline, task_id=run_data_processing, run_id=manual__2024-09-19T21:38:21.389569+00:00, execution_date=20240919T213821, start_date=20240919T213944, end_date=20240919T213946
[2024-09-19T21:39:46.597+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-19T21:39:46.616+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-19T21:39:46.618+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
